.vib/airflow/cypress/cypress.json

```diff
@@ -1,10 +1,9 @@
 {
-    "baseUrl": "http://localhost",
-    "pageLoadTimeout": 240000,
-    "defaultCommandTimeout":80000,
-    
-    "env": {
-      "username": "user",
-      "password": "ComplicatedPassword123!4"
-    }
+  "baseUrl": "http://localhost",
+  "pageLoadTimeout": 240000,
+  "defaultCommandTimeout": 80000,
+  "env": {
+    "username": "user",
+    "password": "ComplicatedPassword123!4"
```

Comment:
It seems like this is not sorted

Reply:
Using `jq --sort-keys .` should solve this.

Reply:
Done